### DISCUSSION TITLE
Install Operating System | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#126] [Q&A]
# Install Operating System | Seeed Studio Wiki

ODYSSEY - X86J4105

https://wiki.seeedstudio.com/linkstar-install-system/

**Comment by oelegeirnaert** [comment]
How to flash it with a Linux computer?
I don't have Windows...

**Comment by oelegeirnaert** [comment]
The TF card option is not working for me.

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.

**Comment by Bedasek** [comment]
I installed OpenWrt and use it as router. Tho the wifi signal strength is pretty week. One narrow brick wall and the signal strength drops down a lot. I assume it's mainly due to the small antennas it came with. I ordered this pair of antenna in hope it will improve it  - https://www.aliexpress.com/item/1005003041434976.html

**Reply by mygittyhubby** [reply]
how't that antenna working for you?

**Reply by Bedasek** [reply]
All I can say is....It works :D 
TBH, I don't know if it made it any better in terms of signal strength ar anything.

**Comment by mygittyhubby** [comment]
I downloaded the R22.11.18 openwrt and flash it to the eMMC and it turn out to be version R22.11.13. so weird, and it has no VLAN function, there's no Service-> OpenVPN menu item

**Reply by TahomaSoft** [reply]
OpenWRT is now supporting the LinkStar H68K in mainline snapshot.

**Reply by Bedasek** [reply]
@TahomaSoft Where do you see that? Tho I found it in **Table of Hardware: Firmware downloads**, it's not showing any fw

**Reply by TahomaSoft** [reply]
Take a look at:
https://firmware-selector.openwrt.org/?version=SNAPSHOT&target=rockchip%2Farmv8&id=hinlink_h66k

**Reply by TahomaSoft** [reply]
Also note that HinLink H6xK and LinkStart H6x are the same.

**Reply by TahomaSoft** [reply]
Also, there's a lot of discussion on these boxes over at forum.openwrt.org that might be of help.
https://forum.openwrt.org/t/wip-support-for-linkstar-h68k-rk3568/233527/60

**Reply by TahomaSoft** [reply]
I've got buildroot running on the LinkStar h68k, with hdmi console activated.
